### PR TITLE
model: add LoadData statement flag

### PR DESCRIPTION
**Pull request by Deardrops**
### What problem does this PR solve? 

add LoadData statement flag in model

### What is changed and how it works?

add `FlagInLoadDataStmt` field in `model/flags.go`

### Check List 

Tests 

 - None

Code changes

 - Has exported variable/fields change

Side effects

- None

Related changes

- https://github.com/pingcap/tidb/pull/11067

### CODE REVIEW

**Comment by kennytm** [comment]
LGTM, but could you please sign the CLA and link to the TiDB PR leading to this addition? Thanks.

**Comment by kennytm** [comment]
LGTM, but could you please sign the CLA and link to the TiDB PR leading to this addition? Thanks.

**Comment by Deardrops** [comment]
@lonng ready for merge, PTAL

File: model/flags.go [hunk 1]
```diff
@@ -42,4 +42,6 @@ const (
 	FlagDividedByZeroAsWarning = 1 << 8
 	// FlagInUnionStmt indicates if this is a UNION statement.
 	FlagInUnionStmt = 1 << 9
+	// FlagInLoadDataStmt indicates if this is a LoadData statement.
```
Comment (lonng) [inline model/flags.go]:
```suggestion
	// FlagInLoadDataStmt indicates if this is a LOAD DATA statement.
```

**Pull request merged**